data/Mapped2Willow.json

```diff
@@ -626,10 +706,22 @@
       "InputDtmi": "dtmi:mapped:core:Fire_Sprinkler_Water_Storage_Tank;1",
       "OutputDtmi": "dtmi:com:willowinc:SprinklerTank;1"
     },
+    {
+      "InputDtmi": "dtmi:mapped:core:Fish_Tank;1",
+      "OutputDtmi": "dtmi:com:willowinc:FishTank;1"
+    },
+    {
+      "InputDtmi": "dtmi:mapped:core:Floor_Buffer;1",
```

Comment:
This is a bit of an odd one. I looked into the specific asset that Walmart had listed as a Floor Buffer and the manufacturer even calls it a Burnisher and has an article about the difference. So basically Walmart's category is wrong.